pkg/agent/testdata/AKSWindows2019+CustomVnet/CustomData

```diff
@@ -0,0 +1,491 @@
+    $MasterIP,
+
+    [parameter()]
+    [ValidateNotNullOrEmpty()]
+    $KubeDnsServiceIp,
+
+    [parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    $MasterFQDNPrefix,
+
+    [parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    $Location,
+
+    [parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    $AgentKey,
+
+    [parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    $AADClientId,
+
+    [parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    $AADClientSecret, # base64
+
+    [parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    $NetworkAPIVersion,
+
+    [parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    $TargetEnvironment,
+
+    [string]
+    $UserAssignedClientID
+)
+
+# These globals will not change between nodes in the same cluster, so they are not
+# passed as powershell parameters
+
+## SSH public keys to add to authorized_keys
+$global:SSHKeys = @( "testsshkey" )
+
+## Certificates generated by aks-engine
+$global:CACertificate = ""
+$global:AgentCertificate = ""
+
+## Download sources provided by aks-engine
+$global:KubeBinariesPackageSASURL = "https://acs-mirror.azureedge.net/kubernetes/v1.19.0/windowszip/v1.19.0-1int.zip"
+$global:WindowsKubeBinariesURL = ""
+$global:KubeBinariesVersion = "1.19.0"
+$global:ContainerdUrl = "https://k8swin.blob.core.windows.net/k8s-windows/containerd/containerplat-aks-test-0.0.8.zip"
+$global:ContainerdSdnPluginUrl = ""
+
+## Docker Version
+$global:DockerVersion = "19.03.11"
+
+## ContainerD Usage
+$global:ContainerRuntime = "docker"
+$global:DefaultContainerdRuntimeHandler = "process"
+$global:HypervRuntimeHandlers = ""
+
+# To support newer Windows OS version, we need to support set ContainerRuntime,
+# HypervRuntimeHandlers and DefaultContainerdRuntimeHandler per agent pool but
+# current code does not support this. Below is a workaround to set contianer
+# runtime variables per Windows OS version.
+#
+# Set default values for container runtime variables for AKS Windows 2004
+if ($([System.Environment]::OSVersion.Version).Build -eq "19041") {
+    $global:ContainerRuntime = "containerd"
+    $global:HypervRuntimeHandlers = "17763,19041"
+    $global:DefaultContainerdRuntimeHandler = "process"
+}
+
+## VM configuration passed by Azure
+$global:WindowsTelemetryGUID = "fb801154-36b9-41bc-89c2-f4d4f05472b0"
+
+$global:TenantId = "tenantID"
+
+$global:SubscriptionId = "subID"
+$global:ResourceGroup = "resourceGroupName"
+$global:VmType = "vmss"
+$global:SubnetName = "subnet1"
+$global:MasterSubnet = "10.240.0.0/16" 
+$global:SecurityGroupName = "aks-agentpool-36873793-nsg"
+$global:VNetName = "aks-vnet-07752737"
+$global:RouteTableName = "aks-agentpool-36873793-routetable"
+$global:PrimaryAvailabilitySetName = ""
+$global:PrimaryScaleSetName = "aks-agent2-36873793-vmss"
+
+$global:KubeClusterCIDR = "10.240.0.0/16"
+$global:KubeServiceCIDR = "10.0.0.0/16"
+$global:VNetCIDR = "10.0.0.0/8"
```

Comment:
customvnet cidr is not the default value.

Reply:
Thanks. Will try customvnet and give a comparison.

Reply:
@AbelHu  Could you please double confirm this? I checked this on customvnet node, and the VNetCIDRis  still "10.0.0.0/8"
```
az aks nodepool add --resource-group qinhao-customvnet --cluster-name qinhao-customvnet     --os-type Windows \
    --name npwin2 --node-count 1  --vnet-subnet-id "/subscriptions/b9d2281e-dcd5-4dfd-9a97-0d50377cdf76/resourceGroups/qinhao-customvnet/providers/Microsoft.Network/virtualNetworks/myVnet/subnets/subnet2" --kubernetes-version  1.17.13

```

Reply:
You are still using ip range in 10.0.0.0/8 so you will not see any difference. You can try other ip address space, for example, `172.17.0.0/16`.

Reply:
We currently support only one vnet,  so the ip range will be `10.0.0.0/8`

Reply:
One vnet does not mean `10.0.0.0/8`. You should be able to create a vnet with `172.17.0.0/16` and use it with AKS

Reply:
Thanks for this, in a another try.

Reply:
Corrected the testdata